src/components/RestaurantRestaButton.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+const RestaurantRestaButton = props => {
+  return (
+    <div>
+      <button onClick={props.reduceOne} className="btn btn-danger mx-2">
+        <span>&#8722;</span>
+      </button>
+    </div>
+  );
+};
+
+export default RestaurantRestaButton;
```

Comment:
This is good passing a function as a prop, and while this is fine for now, once you're in a professional environment make sure you're naming your functions in english (or whatever language of the team you're working in) so that they make sense to other people reading your code!